build.gradle.kts

```diff
@@ -52,7 +52,7 @@ task<Test>("cleanUpAfterBamboo").apply {
 
 dependencies {
     testCompile(project(":virtual-users"))
-    compile(fileTree(mapOf("dir" to "lib", "include" to "*.jar")))
+    testCompile(fileTree(mapOf("dir" to "lib", "include" to "*.jar")))
```

Comment:
What's the difference between compile and testCompile?

Reply:
Found the answer here:https://github.com/atlassian/jira-hardware-exploration/pull/14/files#r272939312